modules/ROOT/pages/asm-secret-group-creation-task.adoc

```diff
@@ -1,27 +1,27 @@
-= Create a Secret Group (Anypoint Platform)
-ifndef::env-site,env-github[]
-include::_attributes.adoc[]
-endif::[]
+= Create and Edit Secret Group (Anypoint Platform)
 
+Secrets Manager enables you to create and edit secrets groups, which is a logical grouping of secrets. Secrets Manager also enables you to create new secret types and add them to a secrets group.
```

Comment:
Out of scope but something to think about: could create a partial here so you're not having to restate what a secret group is.